kernel/kselftest.py.data/trace_tests.yaml

```diff
@@ -1,6 +1,4 @@
 component: !mux
-    ftrace:
-        comp: 'ftrace'
     ptrace:
         comp: 'powerpc'
         subtest: 'ptrace'
```

Comment:
also here run_type of upstream could be added

>     upstream:
>          type: 'upstream'
>          location: 'https://github.com/torvalds/linux/archive/master.zip'

so that trace_tests.yaml could be a generic file, and 'ftrace_powerpc.yaml' or whatever fits best could be specific to powerpc as your need.